app/services/reports/base_report.rb

```diff
@@ -6,11 +6,7 @@ class BaseReport
 
     def fiscal_start_date
       now = Time.zone.now
-      if now.month >= 10
-        now.strftime('10-01-%Y')
-      else
-        "10-01-#{now.year - 1}"
-      end
+      "10-01-#{now.month >= 10 ? now.year : now.year - 1}"
```

Comment:
Re: discussion about formatting, just to restate my opinions from our call:

1. I'd prefer if we did this via object manipulation instead of string interpolation, ex:
    ```ruby
    timestamp = if now.month >= 10
      now.change(month: 10, day: 1)
    else
      now.change(year: now.year - 1, month: 10, day: 1)
    end
    ```

2. use the database adapter to quote/escape/format/convert things for queries for us (I would do this in the method that interpolates the SQL, not here, but same result)
    ```ruby
    ActiveRecord::Base.connection.quote(timestamp)
    => "'2019-10-01 18:14:42.171273'"
    ```

Reply:
and we could also to `.to_date` on there to drop the seconds, etc

```ruby
now = Time.zone.today # Date instance, not a Time instance
ActiveRecord::Base.connection.quote(now)
=> "'2020-03-06'"

```